src/main/resources/db/migration/0300/V348__FundingEntities.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE accelerator.funding_entities
```

Comment:
Maybe bikeshedding, but I wonder if we should create a new schema for these tables rather than putting them under `accelerator`. Functionally there's no difference, of course; it's just about organizing things. My assumption is that we'll be adding lots more funder-related tables over time and it might be nice to have them in a separate namespace from the tables that hold project data.

Reply:
I like that idea and this is probably the best time to do it. Using schema `funder`